src/Header.js

```diff
@@ -3,8 +3,8 @@ import React from 'react';
 class Header extends React.Component {
     constructor(props) {
         super(props);
-        this.restFirstName = ["Robbie's", "Hometown", "Local", "Pretty Decent", "Organic"];
-        this.restSecondName = ["Yummy Stuff", "Inventions", "Food", "Plates", "Experience", "Snacks", "Express"];
+        this.restFirstName = ["Robbie's", 'Hometown', 'Local', 'Pretty Decent', 'Organic'];
```

Comment:
I see you switched to the single quotes, which is good! There's just one set of leftover double quotes here.

Reply:
I was thinking that I would need to leave those as double quotes since I have a single quote in the word. I will try changing to single quotes and using an escape character!